manifests/networkd.pp

```diff
@@ -32,4 +75,35 @@
       notify  => Service['systemd-networkd'],
     }
   }
+
+  $interfaces.each | String[1] $interface_name, Systemd::Interface $interface | {
+    $_filename=pick($interface['filename'], $interface_name)
+    if 'link' in $interface.keys() {
+      systemd::network { "${_filename}.link":
+        path    => $path,
+        content => epp('systemd/network.epp', {
+            fname  => "${_filename}.link",
+            config => deep_merge(pick($link_profiles[$interface_name], {}), $interface['link']),
+        }),
```

Comment:
Better to put the closing braces on separate lines to avoid the need for double indentation:

```suggestion
          fname  => "${_filename}.link",
          config => deep_merge(pick($link_profiles[$interface_name], {}), $interface['link']),
        },
      ),
```

Reply:
I don't think that this makes the code better readable. I will not change that for now.

Reply:
It does. The double indentation that you have to do currently is really just a bug in puppet-lint, I think.

Reply:
no the double intentation is correct since there are to brackets to intent. 
to clarify:
my solution:
```
epp('template, {
    var => val,
})
```
your solution:
```
epp('template,
 {
    var => val,
 }
)
```

Reply:
Yes, IMO it is easier to follow the indentation when there is only at most one level of change between each line (your example has `var => val,` indented one level too far in my solution), easier to match up the opening and closing braces, and reduces diffs when adding and removing items to the comma-separated lists by already having separate lines.

Reply:
I made a test commit with your suggestion. I expect it to fail on validation test ! let's see

Reply:
ok validatio now fails, the it needs 12 spaces, no way to lower that.
I will remove the last commit tomorrow. Think we can resolv this.

Reply:
Right, it's clearly not correct to have the closing brace on the same line as the content inside of the brace. You'd have to also move the opening brace and fix the indentation.